test/services/helpers/services/users.js

```diff
@@ -32,10 +33,36 @@ const createTestUser = (app, opt) => ({
 		discoverable,
 	}))
 	.then((user) => {
-		if (!manualCleanup) {
-			createdUserIds.push(user._id.toString());
+		const consentPromises = [];
+		if (consentStatus === 'parentsAgreed' && roles.includes('student')) {
+			consentPromises.push(app.service('consents').create({
+				userId: user._id,
+				parentConsents: [{
+					privacyConsent: true,
+					termsOfUseConsent: true,
+				}],
+			}));
+		} else if (consentStatus === 'ok') {
+			consentPromises.push(app.service('consents').create({
```

Comment:
@Metauriel filters seems not to set something.

Reply:
> @josafatburmeister filters seems not to set something.

Could you please elaborate? Where should be set something?